src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using SFA.DAS.Assessors.InnerApi.Responses;
+
+namespace SFA.DAS.Assessors.Api.Models
+{
+    public class GetCourseListItem
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public int Level { get; set; }
+        public int MaxFunding { get; set; }
+        public int TypicalDuration { get; set; }
+        public StandardDate StandardDates { get; set; }
+
+        public static implicit operator GetCourseListItem(GetStandardsListItem source)
+        {
+            return new GetCourseListItem
+            {
+                Id = source.Id,
+                Title = source.Title,
+                Level = source.Level,
+                MaxFunding = source.MaxFunding,
+                TypicalDuration = source.TypicalDuration,
+                StandardDates = source.StandardDates
```

Comment:
Need `IsActiveStandards` adding as well - its this logic:

```
public static bool IsStandardActive(GetStandardsListItem standard)
        {
            return standard.StandardDates.EffectiveFrom.Date <= DateTime.UtcNow.Date
                   && (!standard.StandardDates.EffectiveTo.HasValue ||
                       standard.StandardDates.EffectiveTo.Value.Date >= DateTime.UtcNow.Date);
        }
        
```
Which is in the recruit outer api but might want moving onto that standard response in shared